modules/channel-twilio/src/renderers/card.ts

```diff
@@ -0,0 +1,29 @@
+import * as sdk from 'botpress/sdk'
+import { ChannelRenderer } from 'common/channel'
+import { CHANNEL_NAME } from '../backend/constants'
+import { TwilioContext } from '../backend/typings'
+
+export class TwilioCardRenderer implements ChannelRenderer<TwilioContext> {
+  get channel(): string {
+    return CHANNEL_NAME
+  }
+
+  get priority(): number {
+    return -1
+  }
+
+  get id(): string {
+    return TwilioCardRenderer.name
+  }
+
+  handles(context: TwilioContext): boolean {
+    return context.payload.type === 'card'
+  }
+
+  render(context: TwilioContext) {
+    const payload = context.payload as sdk.CardContent
+
+    // we convert our card to a carousel
+    context.payload = context.bp.experimental.render.carousel(payload)
```

Comment:
why ? I guess that's necessary for Twillio

Reply:
We have this same renderer on all channel. Basically what I did there is move the logic of converting a card to a carousel from the card content renderer to this channel renderer. So we get the same result (being that a card eventually becomes a carousel) but since it's not done in the content renderer, it allows us to do something like this : `bp.messages.send(convoId, bp.render.card(...))`